### PR TITLE
"Profile not set" pop-up

### PR DESCRIPTION
**Pull request by sergiuilie**
fix: "Profile not set" pop-up #270

### CODE REVIEW

**Comment by sergiuilie** [comment]
tested on VSCode and CHE

File: clients/cobol-lsp-vscode-extension/src/extension.ts [hunk 1]
```diff
@@ -48,7 +48,8 @@ export async function activate(context: vscode.ExtensionContext) {
 
     // Listeners
     context.subscriptions.push(vscode.workspace.onDidChangeConfiguration(event => {
-        if (event.affectsConfiguration(SETTINGS_SECTION)) {
+        if (event.affectsConfiguration(SETTINGS_SECTION) &&
+        vscode.workspace.getConfiguration().get(SETTINGS_SECTION + ".profiles")) {
```
Comment (zacanbrcom) [inline clients/cobol-lsp-vscode-extension/src/extension.ts:52]: Couple of questions: 
- can we use ```vscode.workspace.getConfiguration(SETTINGS_SECTION).get("profiles")```?
- is mandatory have the ```.``` before profile?

Comment (sergiuilie) [reply clients/cobol-lsp-vscode-extension/src/extension.ts:52]: good idea, I will modify it

**Pull request merged**